app/controllers/api/messages_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Api
+  class MessagesController < ApplicationController
+    def random_greeting
+      greetings = %w[سلام Hi Hola こんにちは Ciao]
+      render json: { greeting: greetings.sample }
+    end
```

Comment:
- [ ] In this controller method, you are just requiring a hard coded message which is not meant to be so. you are meant to query the database through the model and return a random message. :+1: You can make use of the solution you seem to have used in the other controller
```ruby
 def random_greeting
      greeting = Message.order('RANDOM()').first
      render json: { greeting: greeting }
   end
```
or you can take this approach
```ruby
  def index
    rand_id = rand(1..Greeting.count)
    @message = YourModelNameHere.find(rand_id) if rand_id

    if @message
      render json: @message, status: :ok
    else
      render json: { message: 'No Greeting found' }, status: :not_found
    end
  end
```